app/controllers/reviews_controller.rb

```diff
@@ -5,4 +5,8 @@ def new
   def edit
     @review = Review.find(params[:id])
   end
+
+  def destroy
+     
+  end
```

Comment:
Not needed, you cant navigate to a delete path and you already have this in your api. This just exposes a dead end in your app. Remove.